README.md

```diff
@@ -137,12 +137,20 @@ If you'd prefer to set authentication values manually in your code, the SDK supp
 
 ### IAM
 
-IBM Cloud is migrating to token-based Identity and Access Management (IAM) authentication. IAM authentication uses a service API key to get an access token that is passed with the call. Access tokens are valid for approximately one hour and must be regenerated.
+IBM Cloud has migrated to token-based Identity and Access Management (IAM) authentication. IAM authentication uses a service API key to get an access token that is passed with the call. Access tokens are valid for approximately one hour and must be regenerated.
 
 You supply either an IAM service **API key** or an **access token**:
 
 - Use the API key to have the SDK manage the lifecycle of the access token. The SDK requests an access token, ensures that the access token is valid, and refreshes it if necessary.
 - Use the access token if you want to manage the lifecycle yourself. For details, see [Authenticating with IAM tokens](https://console.bluemix.net/docs/services/watson/getting-started-iam.html).
+- For untrusted environments use a server-side token generator to generate access tokens for your environment using your IAM API key. The generated access tokens will be valid for one hour and can be refreshed.
```

Comment:
Maybe add the following description?
```
If you want to generate access tokens for your environment using your IAM API key. The generated access tokens will be valid for one hour and can be refreshed.
```

Reply:
Perhaps the sentence should be more like the other suggestions for usage, so how about:

> Use a server-side to generate access tokens using your IAM API key for untrusted environments like client-side scripts. The generated access tokens will be valid for one hour and can be refreshed.


What do think about this?

Reply:
That sounds good